tests/shell/shell_recorder/tutorial_wrapper/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ if (ENABLE_ASAN)
 	message (STATUS "Excluding Markdown Shell Recorder test for `validation`, as it leaks memory and fails with ASAN enabled")
 else (ENABLE_ASAN)
 	add_msr_test (tutorial_validation "${CMAKE_SOURCE_DIR}/doc/tutorials/validation.md" REQUIRED_PLUGINS ni validation)
+	add_msr_test (tutorial_crypto "${CMAKE_SOURCE_DIR}/doc/tutorials/crypto.md" REQUIRED_PLUGINS crypto_gcrypt fcrypt)
```

Comment:
Unfortunately, add_msr_test currently does not support dependencies to tools (which would be required here). It should be fairly easy to add, though.